scirepeval.py

```diff
@@ -127,6 +127,7 @@ def evaluate(self, model: Union[Model, List[Model]], output: str):
     parser.add_argument('--adapters-dir', help='path to the adapter checkpoints', default=None)
     parser.add_argument('--fusion-dir', help='path to the fusion checkpoints', default=None)
     parser.add_argument('--adapters-chkpt', help='hf adapter names keyed on tasks', default=None, type=json.loads)
+    parser.add_argument('--pooling-mode',  default="cls", help='Pooling mode to get embeddings from encoder, must be one of "cls" or "mean"')
```

Comment:
Plz change the default to None. Cls is a bit confusing since for the control code approach we pick the corresponding control code token rather than cls.